ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -135,7 +135,8 @@
     # Note - had to have these set to `install-mock` to progress role
     # installation before changing config to restored DB from other system.
     - role: openmicroscopy.omero-server
-      omero_server_release: 5.3.1
+      omero_server_release: 5.3.2
+      omero_server_upgrade: True
```

Comment:
Which - the 'upgrade' flag? I'm not sure it does any harm if we're controlling the version to be installed with the `_release` variable. That flag has existed for some time in the ns-web playbook (inserted at the 5.3.0->5.3.1 upgrade: ﻿﻿﻿https://github.com/openmicroscopy/infrastructure/pull/260/files#diff-73638a117d0afc35645cb554e59abdc1R48).